float64[]  values
float64    time
float64    phase_val
TwoDArray  sigma
string[]   condition_keys